tests/001-test-use-devs-to-create-thin-pool.sh

```diff
@@ -17,6 +17,7 @@ test_devs() {
   cat << EOF > /etc/sysconfig/docker-storage-setup
 DEVS="$devs"
 VG=$vg_name
+CONTAINER_THINPOOL=docker-pool
```

Comment:
@lsm5 this is not a problem as long as CONTAINER_THINPOOL=docker-pool.  So when we update docker.service to using container-storage-setup with inputfile/outputfile, we just need to make sure we add 
CONTAINER_THINPOOL=docker-pool
In docker-storage-setup file we ship.